impl-endpoints/cloudflare-go/setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
This looks to be the same script as `impl-endpoints/boringssl/setup.sh`. Any way to deduplicate this?